az-core/src/main/java/azkaban/Constants.java

```diff
@@ -884,4 +884,12 @@ public static class FlowParameters {
     // Constant to define the strategy to restart the execution, default to "restart_from_root",
     public static final String FLOW_PARAM_RESTART_STRATEGY = "flow.retry.strategy";
   }
+
+  public enum FlowRetryStrategy {
```

Comment:
Should we validate these values in HttpRequestUtils class when a user submits a new execution? If addressed in a separate PR could you please add a TODO comment in the code?

Reply:
good call, let me validate them in this PR